modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISchemaRelevantObjectEntryModelListenerTest.java

```diff
@@ -85,6 +103,18 @@ public void test() throws Exception {
 			"An API schema must be related to an API application.",
 			jsonObject.get("title"));
 
+		JSONObject apiSchemaJSONObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
I think that you can reuse the apiSchema at line `168`, because they are the same. As well as you can delete `apiApplicationPublishedJSONObject`.

Reply:
will check that! thanks!